psth_functions/graph_PSTH.m

```diff
@@ -54,7 +55,8 @@
             end
 
             %% Creating the PSTH graphs
-            for neuron = 1:total_region_neurons
+            for neuron_iter = 1:total_region_neurons
+                neuron = find(label_log.(current_region).user_channels == neuron_iter);
```

Comment:
A flag was supposed to be added here to allow the users to choose if the plotting order should be controlled by the user channels column retrieved from the labels file. This is particularly important since this check assumes that the user channels column will be numerical whereas the user channels column in the labels file does not enforce a specific type.